boards/imix/src/main.rs

```diff
@@ -374,7 +374,8 @@ pub unsafe fn reset_handler() {
     let button = components::button::ButtonComponent::new(board_kernel).finalize(
         components::button_component_helper!((
             &sam4l::gpio::PC[24],
-            capsules::button::GpioMode::LowWhenPressed
+            capsules::button::GpioMode::LowWhenPressed,
+            kernel::hil::gpio::FloatingState::PullUp
```

Comment:
```suggestion
            kernel::hil::gpio::FloatingState::PullNone
```

imix has a pull up on board as well.